### PR TITLE
[OBSDEF-4083][OBSDEF-8020][OBSDEF-8295] UI log forwarding, 2 replicas for gui, New objectstore wiz & repl on by default.

### PR DESCRIPTION
**Pull request by maheshkulkarni4**
## Purpose
[OBSDEF-4083](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-4083 )
OBSDEF-4083 - vsphere plugin: add support for log forwarding

[OBSDEF-8295](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-8295 )
OBSDEF-8295 - Update Replica counts for plugin UI and Graphql Deploument

[OBSDEF-8020](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-8020 )
OBSDEF-8020 - Replication page should get enabled without enabling the flag.

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [x] vSphere7 make package Passed
- [x]  Installtion on vSphere7 Passed
- [x]  Installation on Openshift Passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [x] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

### results from charts-custom-ci 
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/charts-ci-vsph7-large/571/

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: objectscale-portal/templates/objectscale-portal-deployment.yaml [hunk 1]
```diff
@@ -76,3 +76,11 @@ spec:
             - name: certificate
               mountPath: /etc/nginx/ssl
             {{- end }}
+      {{- if not .Values.logReceiver.type }}
+{{ include "common-monitoring-lib.rsyslog_client_sidecar_stdout" . | indent 8 }}
+      {{- end }}
+      {{- if not .Values.logReceiver.type }}
```
Comment (ben-schumacher) [inline objectscale-portal/templates/objectscale-portal-deployment.yaml:82]:
```suggestion
```

Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: Resolved.

**Review by vgupta-mickey — APPROVED** [review]

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
the changes don't look right. 

from @vazhem  we simply need to add 3 labels to the pod.

File: objectscale-portal/values.yaml [hunk 2]
```diff
@@ -89,13 +89,37 @@ rsyslog-client:
...
+  # host: my-syslog-server
+  #
+  # Configure protocol to receive logs for a Syslog-based receiver. Valid values
+  # for syslog receivers are "tcp". Valid values for Elasticsearch are
+  # "http" and "https"
+  # protocol: tcp
+  #
+  # Port to use for customer provided Syslog or Elasticsearch receiver
+  # port: 514
+#
```
[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/values.yaml]: yeah.. without this 
{{- if not .Values.logReceiver.type }} 
fails.

File: objectscale-portal/templates/objectscale-portal-deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: This is adding the rsyslog sidecar container... @victor-polyansky  mentioned it on the Ticket too and confirmed that this is needed 
https://jira.cec.lab.emc.com/browse/OBSDEF-4083

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: objectscale-portal/values.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (victor-polyansky) [reply objectscale-portal/values.yaml]: I guess this `logReceiver` config is some old configuration for logging, it's not used in the objectscale logging now and can be removed. Checks for `{{- if not .Values.logReceiver.type }}` can be removed too.

File: objectscale-portal/templates/objectscale-portal-deployment.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ metadata:
     operator: objectscale-operator
     product: objectscale
 spec:
-  replicas: 1
+  replicas: 2
```
[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml:19]: Yes

**Review by vazhem — APPROVED** [review]

File: objectscale-portal/templates/objectscale-portal-deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-deployment.yaml]: This is removed now as discussed.

File: objectscale-portal/values.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (maheshkulkarni4) [reply objectscale-portal/values.yaml]: This is removed as well.

**Comment by eddiepavkovic** [comment]
Did the labels actually get added. Look at my pr #821    you may need a change to values.yaml

**Review by eddiepavkovic — APPROVED** [review]
ok.  after a bunch of merges and tests its working better now.
approving and merging.

**Pull request merged**